crates/net/network/src/metrics.rs

```diff
@@ -244,3 +246,45 @@ pub struct EthRequestHandlerMetrics {
     /// Number of received bodies requests
     pub(crate) received_bodies_requests: Counter,
 }
+
+/// Eth67 announcement metrics, track entries by TxType
+#[derive(Metrics)]
+#[metrics(scope = "network.transaction_fetcher")]
+pub struct AnnouncedTxSizeMetrics {
+    /// Histogram for tracking legacy transaction sizes
```

Comment:
docs here should be updated too to reflect what the code is actually doing
```suggestion
    /// Histogram for tracking frequency of legacy transaction type
```